### ISSUE TITLE
Add IPv6

### ISSUE THREAD

**Issue by NghiaTranUIT** [ProxymanApp/Proxyman#1355]
## Description
Some users request that Proxyman should listen on IPv6 too. We should support it.

**Comment by NghiaTranUIT** [comment]
Beta build: https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_3.9.0_IPv6_on_Preference_General.dmg

### Screenshot
<img width="712" alt="Screen_Shot_2022-09-09_at_13_59_47" src="https://user-images.githubusercontent.com/5878421/189292847-2d87fa52-1edb-4070-b428-e960b81e01b8.png">

**Comment by king112ola** [comment]
Hi, I am testing on my ios remote device (connected to proxyman via wifi manual proxy setting) when I make a direct HTTPS request in Chrome browser using the ipv4 address e.g. https://1.1.1.1 , proxyman was able to capture that, but when I test an app that makes a similar request to (ipv4 - 142.250.198.170:443) or (ipv6 - [2404:6800:4005:815::200a]:443), I couldn't see it on proxyman.

Is that a problem with the ios wifi manual proxy? or I have done anything wrong.

**Comment by NghiaTranUIT** [comment]
@king112ola may I ask if your iOS is a native app or React Native?

and does your connection via IP success? Or failed?

**Comment by king112ola** [comment]
@NghiaTranUIT, thanks for your reply!

The app isn't developed by me, it's a publicly available app on the Apple IOS App Store. What I wanted to achieve was to capture all the network traffic from this app and route it through a new app I developed, using the Network Extension Relay Manager's [matchDomains](https://developer.apple.com/documentation/networkextension/nerelaymanager/matchdomains) to bypass the app's regional blocking. However, the issue is that the domains I captured using Proxyman (no IPv4/IPv6 addresses found, just domains) were insufficient to bypass the restriction.

Interestingly, when I used [excludedDomains](https://developer.apple.com/documentation/networkextension/nerelaymanager/excludeddomains) (similar to the way a typical VPN works) to route all network traffic to my relay without relying on matchDomains, it worked. This led me to wonder whether Proxyman, when used with an iOS remote device (via Wi-Fi manual proxy settings), might not capture certain hidden types of network traffic, such as IPv4, gRPC, or WebSocket.

**Comment by NghiaTranUIT** [comment]
From what I know, If you manually set up the Proxy via WiFi, some apps might not respect the HTTP Proxy. For example, react native app has their own networking library, and it's usually bypass the HTTP Proxy.

However, if you're using Proxyman VPN (iOS app), all traffic will be navigate via the VPN and you might capture it.

Can you try using Proxyman for iOS and test again?

**Comment by king112ola** [comment]
> From what I know, If you manually set up the Proxy via WiFi, some apps might not respect the HTTP Proxy. For example, react native app has their own networking library, and it's usually bypass the HTTP Proxy.
> 
> However, if you're using Proxyman VPN (iOS app), all traffic will be navigate via the VPN and you might capture it.
> 
> Can you try using Proxyman for iOS and test again?

I have just tested it on the iOS version, but I only see HTTP domain requests. After some testing on wireshark (ios using rvictl), I suspect this is due to a DNS leak. Am I correct in assuming that Proxyman cannot capture DNS resolution requests?

<img width="498" alt="Image" src="https://github.com/user-attachments/assets/82ac6bc0-b43b-4066-aa60-2641b5640f7d" />

**Comment by NghiaTranUIT** [comment]
you're right. Proxyman can't capture DNS because it operates over TLS layer.

Proxyman only works with application layers, HTTP/HTTPS and Websocket

**Comment by king112ola** [comment]
Got it, thx for your reply!
